test/jdk/javax/swing/JScrollBar/4865918/bug4865918.java

```diff
@@ -44,30 +42,23 @@
 public class bug4865918 {
 
     private static TestScrollBar sbar;
-    private static JFrame frame;
     private static final CountDownLatch mousePressLatch = new CountDownLatch(1);
 
     public static void main(String[] argv) throws Exception {
-        try {
-            Robot robot = new Robot();
-            SwingUtilities.invokeAndWait(() -> createAndShowGUI());
-
-            robot.waitForIdle();
-            robot.delay(1000);
-
-            SwingUtilities.invokeAndWait(() -> sbar.pressMouse());
-            if (!mousePressLatch.await(2, TimeUnit.SECONDS)) {
-                throw new RuntimeException("Timed out waiting for mouse press");
-            }
+        String osName = System.getProperty("os.name");
+        if (osName.toLowerCase().contains("os x")) {
+            System.out.println("This test is not for MacOS, considered passed.");
```

Comment:
Test is restricted to run on non-mac systems.
Is it required to check OS explicitly here ?

Reply:
For standalone run..